cobc/ChangeLog

```diff
@@ -1,4 +1,23 @@
 
+2023-10-17  David Declerck <david.declerck@ocamlpro.com>
+
+	BUG #923: generated modules init/clear unused decimal constants
+	* tree.h (struct cb_program): new `decimal_constant` field to store
+	  decimal constants used by the current program
+	* codegen.c (literal_list): removal of the unused `c` field
```

Comment:
that's likely "x", not "c"
rechecking the Changelog show that we don't use backticks in there, so please adjust the entries